components/script/dom/websocket.rs

```diff
@@ -273,7 +270,7 @@ impl WebSocket {
                     WebSocketEvent,
                     task,
                     Some(pipeline_id),
-                    TaskSourceName::Networking,
+                    WebsocketTaskSource::NAME,
```

Comment:
Is it better to use `TaskSourceName::Websocket`?
Maybe it would be great to keep consistency like other `task_canceller` to use `TaskSourceName` enum!